<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Hide">
     <attribute name="title">
      <string>Hide</string>
     </attribute>
     <widget class="QGraphicsView" name="imageGraphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="openImageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EncodeButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Hide</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveAsButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>310</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save As</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="toHideTextEdit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>171</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Decode">
     <attribute name="title">
      <string>Decode</string>
     </attribute>
     <widget class="QTextEdit" name="toRevealTextEdit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>171</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="imageGraphicsViewDecode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="DecodeButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>310</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Decode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openImageButtonDecode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Image</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSteaganographie">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuSteaganographie"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
